fvm/evm/stdlib/contract.cdc

```diff
@@ -68,7 +68,12 @@ contract EVM {
         /// the address provided in the contractAddress field.
         /// in case of revert, the smart contract custom error message
         /// is also returned here (see EIP-140 for more details).
-        returnedData: String
+        returnedData: String,
+        /// captures the input and output of the calls to the extra precompiled 
+        /// contracts (e.g. Cadence Arch) during the transaction execution.
+        /// This data helps to replay the transactions without the need to
+        /// have access to the full cadence state data.
+        encodedPrecompiledCalls: String
```

Comment:
very nit: `precompiledCalls` them being encoded is not reflected in other names (eg. payload is also encoded above but just called payload), better to be consistent with naming.

Reply:
updated the name, but also moved away from string, and used byte array